distribution/docs/src/main/jdocs/content/_tables/conf-org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin-table-contents.adoc

```diff
@@ -0,0 +1,8 @@
+:type: documentation
+:status: published
+
+|<<org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin,Default Security Attribute Values Plugin>>
+|org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin
+|Mapping of System High attributes to use in case metacards are ingested without any security marking.
```

Comment:
That is a good question. Per https://github.com/codice/alliance/blob/master/catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/main/resources/OSGI-INF/metatype/metatype.xml#L17, it only calls out metacards.